packages/quantic/force-app/main/default/lwc/quanticResult/quanticResult.js

```diff
@@ -71,4 +75,19 @@ export default class QuanticResult extends LightningElement {
     }
     return defaultTemplate;
   }
+
+  get resultPreviewShouldNotBeAccessible() {
+    if (!this.openPreviewId) {
+      return false;
+    }
+    if (this.result.uniqueId !== this.openPreviewId) return true;
+    return false;
```

Comment:
```suggestion
    return this.openPreviewId && (this.result.uniqueId !== this.openPreviewId);
```

Seems like this can be simplified to this, but I don't really understand the condition :)

Reply:
sure yeah! I'll simplify this.

Reply:
I will try to explain the logic behind this as well:
Here we are dealing with the Quantic Result component, its template looks like this:
```
<c-quantic-result>
   <div>Result title</div>
   <div>Result excerpt</div>
   <c-quantic-result-quickview></c-quantic-quickview>
</c-quantic-result>
```

Another useful to know about the QuanticResult component is that it has a property named `openPreviewId`, this property is set by the QuanticResultList on each QuanticResult.
this property behaves as follows:
- if there is NOT any quickview opened: `openPreviewId===null`
- if there a quickview opened: `openPreviewId==='RESULT_ID_OF_THE_PREVIEWD_RESULT'`


1. First, lets focus on the title and the the excerpt, these elements need to be removed from the accessibility when the quickview is open. To achieve this I created the property `isAnyPreviewOpen` which is basically :

```
get isAnyPreviewOpen(){
   return !!this.openPreviwId
}
```

2.  Secondly, we need to handle the accessibility of the `c-quantic-result-quickview`, this element:
- should be accessible when there is not any opened quickview (`openPreviewId===null`)
- should NOT be accessible when a quickview of another result os opened(`openPreviewId==='RESULT_ID_OF_ANOTHER_RESULT'`)
- should be accessible when the quickview of this particular result is opened(`openPreviewId==='RESULT_ID_OF_THIS_PARTICULAR_RESULT'`)

To translate these three points I we have this property `resultPreviewShouldNotBeAccessible` which is basically:
```
get resultPreviewShouldNotBeAccessible(){
   return this.openPreviwId && (this.result.uniqueId !== this.openPreviewId)
}
```